bigquery_etl/cli/view.py

```diff
@@ -197,9 +197,10 @@ def publish(
     views = [View.from_file(f) for f in view_files]
     views = [v for v in views if not user_facing_only or v.is_user_facing]
 
-    with Pool(parallelism) as p:
+    with ThreadPool(parallelism) as p:
```

Comment:
It appears that `ThreadPool` is actually a little faster than `Pool`. This is also what we used previously